### PR TITLE
Update the Wallet Adapter section

### PR DESCRIPTION
**Pull request by jmintuitive**
### Description

This updates the guides for building wallet adapter plugins, as well as the main navigation page. It does not include the `Wallet Standard` page.

### Checklist

- Do all Lints pass?
  - [ ] Have you ran `pnpm spellcheck`?
  - [x] Have you ran `pnpm fmt`?
  - [x] Have you ran `pnpm lint`?

### CODE REVIEW

File: apps/nextra/pages/en/build/sdks/wallet-adapter/dapp.mdx [hunk 1]
```diff
@@ -1,206 +1,474 @@
...
-```
+The legacy standard required installing plugins manually. For wallets that have not updated to using the AIP-62 standard, their plugins must be installed and passed in to the `Provider` manually.
 
-Import the `AptosWalletAdapterProvider`:
+For example:
 
-```jsx
-import { AptosWalletAdapterProvider } from "@aptos-labs/wallet-adapter-react";
+```bash filename="Terminal"
+npm install @bitget-wallet/aptos-wallet-adapter
```
Comment (0xmaayan) [inline apps/nextra/pages/en/build/sdks/wallet-adapter/dapp.mdx:34]:
```suggestion
npm install @pontem/wallet-adapter-plugin
```

File: apps/nextra/pages/en/build/sdks/wallet-adapter/dapp.mdx [hunk 2]
```diff
@@ -1,206 +1,474 @@
...
+  // Update this with a non-manual check to verify whether the network change is a supported feature.
+  // Or have error handling if a network change is attempted for a wallet that does not support the feature.
+  const isNetworkChangeSupported = wallet?.name === "Nightly";
+
+  const isValidNetworkName = () => {
+    return network && Object.values<string>(Network).includes(network.name);
+  };
+
+  return (
+    <div style={{ display: 'flex', flexDirection: 'column', gap: '16px' }}>
```
Comment (0xmaayan) [inline apps/nextra/pages/en/build/sdks/wallet-adapter/dapp.mdx:371]: can we clean this example? the style/css is to noisy and prevent focusing on what is really important

File: apps/nextra/pages/en/build/sdks/wallet-adapter.mdx [hunk 3]
```diff
@@ -2,134 +2,31 @@
...
-Wallets looking to integrate with the adapter should implement their own wallet
-plugin, to ease the process we provide you with a pre-made class that implements
-the basic functionality needed (according to the wallet standard).
+Follow one of these guides for how to implement a Wallet Adapter plugin that dapps can connect to:
 
-The [Wallet Adapter Plugin Template repo](https://github.com/aptos-labs/wallet-adapter-plugin-template)
-holds a pre-made class, a test file, and some config files to help you build and
-publish the plugin as an NPM package.
+1. For [Browser Extension Wallets](./wallet-adapter/browser-extension-wallets.mdx) (ex. [Petra](https://chromewebstore.google.com/detail/petra-aptos-wallet/ejjladinnckdgjemekebdpeokbikhfci?hl=en))
+2. For [SDK Wallets](./wallet-adapter/wallets.mdx) (ex. [T Wallet](https://www.npmjs.com/package/@atomrigslab/aptos-wallet-adapter))
```
Comment (0xmaayan) [inline apps/nextra/pages/en/build/sdks/wallet-adapter.mdx:22]:
```suggestion
2. For [SDK Wallets](./wallet-adapter/wallets.mdx) (ex. [AptosConnect](https://aptosconnect.app/dashboard/main-account/tokens/))
```

File: apps/nextra/pages/en/build/sdks/wallet-adapter/dapp.mdx [hunk 4]
```diff
@@ -1,206 +1,474 @@
...
-  signMessage,
-} = useWallet();
-```
+It is recommended to:
 
-Finally, use
-the [examples](https://github.com/aptos-labs/aptos-wallet-adapter/tree/main/packages/wallet-adapter-react#examples)
-on the package README file to build more functionality into your dapps.
+1. Set `autoConnect` to `true`. 
+2. Include `plugins` for any wallets that are using the legacy adapter.
```
Comment (0xmaayan) [inline apps/nextra/pages/en/build/sdks/wallet-adapter/dapp.mdx:53]:
```suggestion
2. Include `plugins` for any wallets that are using the legacy standard.
```

File: apps/nextra/pages/en/build/sdks/wallet-adapter/dapp.mdx [hunk 5]
```diff
@@ -1,206 +1,474 @@
...
 
-Import the installed wallets:
+<Callout type="info">
+The more modern [AIP-62](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-62.md) wallets do NOT require installing a package - they work by default!
+</Callout>
 
-```jsx
-import { SomeWallet } from "some-wallet-plugin-wallet-adapter";
-```
+The legacy standard required installing plugins manually. For wallets that have not updated to using the AIP-62 standard, their plugins must be installed and passed in to the `Provider` manually.
```
Comment (0xmaayan) [inline apps/nextra/pages/en/build/sdks/wallet-adapter/dapp.mdx:29]: can we have it in a nice info box?

Comment (jmintuitive) [reply apps/nextra/pages/en/build/sdks/wallet-adapter/dapp.mdx]: Moved the legacy standard note into the above info box. The note about providing plugins manually is part of the steps though so I kept those in the "main" text area as they are instructions.

**Review by hariria — APPROVED** [review]

**Comment by hariria** [comment]
Will let @0xmaayan take another pass

**Review by 0xmaayan — APPROVED** [review]

**Pull request merged**